explorer/main.cpp

```diff
@@ -117,8 +118,34 @@ auto ExplorerMain(int argc, char** argv, void* static_for_main_addr,
 
   if (!trace_file_name.empty()) {
     // Adding allowed phases in the trace_stream
```

Comment:
```suggestion
    // Adding allowed phases in the trace_stream.
```

Not your comment, but I can't unsee it, and github lets me suggest. :)